src/pages/member/subcomponents/EventCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import withStyles from '@material-ui/core/styles/withStyles'
+import Typography from '@material-ui/core/Typography'
+import Card from '@material-ui/core/Card'
+import { COLOR } from '../../../constants/Constants'
+
+const styles = {
+    page: {
+        width: '345px',
+        marginLeft: '34px',
+        marginTop: '27px',
+    },
+    title: {
+        color: COLOR.TITLE_GREEN,
+        fontWeight: 'bold',
+        fontSize: '36px',
```

Comment:
does mui support fontsizes in the theme?

Reply:
just tried it on mui typography root and didn't seem to work

Reply:
Oh, can we use typography elements like `h2` `h3` when possible?